chromadb/test/conftest.py

```diff
@@ -15,7 +15,7 @@
 
 hypothesis.settings.register_profile(
     "dev",
-    deadline=30000,
+    deadline=None,
```

Comment:
Do we really want to do this unilaterally everywhere?

Reply:
Do we know why we are having timeout hiccups across the tests? Seems to be happening in js too

Reply:
No idea why it's happening.

This was kind of a change borne of desperation of just trying to get it to pass, but the JS tests don't use Hypothesis so I think there's something else going on. I'll go ahead and revert until we have a more principled reason to change it.